keytar/index.d.ts

```diff
@@ -31,9 +31,9 @@ export declare function addPassword(service: string, account: string, password:
  * @param service The string service name.
  * @param account The string account name.
  *
- * @returns the string password or null on failures.
+ * @returns true on success, false on failure
  */
-export declare function deletePassword(service: string, account: string): string;
+export declare function deletePassword(service: string, account: string): boolean;
```

Comment:
I took a shot at this in 1753b8c0c to validate the return values for all these methods. Let me know how that feels.